source/installing-with-npm/index.html.md.erb

```diff
@@ -19,6 +19,8 @@ weight: 10
 
 3. Install [Dart Sass](https://www.npmjs.com/package/sass) - version 1.0.0 or higher.
 
+    If you are using Dart Sass v1.33.0 or greater, you may see deprecation warnings when compiling your Sass. You can [silence these deprecation warnings](/importing-css-assets-and-javascript/#silence-deprecation-warnings-in-dart-sass) if you need to.
```

Comment:
```suggestion
    If you're using Dart Sass 1.33.0 or greater, you may see deprecation warnings when compiling your Sass. You can [silence these deprecation warnings](/importing-css-assets-and-javascript/#silence-deprecation-warnings-in-dart-sass) if you need to.
```

Do we need call out that it silences warnings caused by dependencies?

Also, is it a 'greater' or a 'higher' version?

Reply:
> Why would users need to - because the warnings are an unwelcome distraction?

Pretty much, yes. They're noise, as they're not able to do anything about them, and they may get mixed up with warnings about code that is in their control that might otherwise be missed.